builds/org.eclipse.n4js.product.build/N4JS__IDE.launch

```diff
@@ -49,8 +49,6 @@
 <setEntry value="org.eclipse.n4js.dependencies.ui.sdk:default"/>
 <setEntry value="org.eclipse.n4js.jsdoc2spec.sdk:default"/>
 <setEntry value="org.eclipse.n4js.lang.sdk:default"/>
-<setEntry value="org.eclipse.n4js.n4jsx.sdk:default"/>
-<setEntry value="org.eclipse.n4js.n4jsx.xpect.sdk:default"/>
```

Comment:
why remove N4JSX from the launch configuration? Instead, I'd expect N4IDL to be added.

Reply:
This is an automatic change that is caused by editing the launch configuration. These entries were removed by the UI, since the features "org.eclipse.n4js.n4jsx.sdk" and "org.eclipse.n4js.n4jsx.xpect.sdk" do not exist anymore.